renpy/display/position.py

```diff
@@ -186,7 +186,7 @@ def compute_raw(value: Position, room: float) -> "int | float | absolute":
         """
 
         if isinstance(value, position):
-            return value.relative * room + value.absolute
+            return float(value.relative * room + value.absolute)
```

Comment:
Doesn't this defeat the point of compute_raw, which is to avoid the cost of a type conversion here?